resources/com/mesosphere/global/terraform_file_deploy.sh

```diff
@@ -177,6 +177,99 @@ EOF
   else
     echo -e "\e[32m nginx reached \e[0m"
   fi
+  echo -e "\e[34m deploying dotnet-sample \e[0m"
+  dcos marathon app add <<EOF
+{
+  "labels": {
+    "HAPROXY_DEPLOYMENT_GROUP": "dotnet-sample",
+    "HAPROXY_0_REDIRECT_TO_HTTPS": "true",
+    "HAPROXY_GROUP": "external",
+    "HAPROXY_DEPLOYMENT_ALT_PORT": "10005",
+    "HAPROXY_0_VHOST": $(terraform output public-agents-loadbalancer)
```

Comment:
just pick any vhost name and do `curl -H "Host: <my-vhost-name>"` no need for changing ports.

Reply:
fixed at https://github.com/dcos-terraform/jenkins-library/pull/10/commits/5f3624c996c85b4ec6076f6b9d193eb8be2e100b